types/webextension-polyfill/namespaces/tabs.d.ts

```diff
@@ -700,7 +700,7 @@ export namespace Tabs {
          * The CookieStoreId used for the tab.
          * Optional.
          */
-        cookieStoreId?: string;
+        cookieStoreId?: string[] | string;
```

Comment:
Confirm that this "Id" property might actually be multiple IDs?

Reply:
Yes, this is a filter attribute, so you can enter either a string or a list of strings:  https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/query